src/status_im/router/core.cljs

```diff
@@ -229,7 +229,11 @@
       (cb {:type handler :community-id (:community-id route-params)})
 
       (= handler :community)
-      (cb {:type handler :community-id (:community-id route-params)})
+      (cb {:type         (if (string/starts-with? (:community-id route-params) "z")
```

Comment:
I think the `handle-uri` function shouldn't have the responsibility of knowing details such as if the community-id starts with `z`. Example refactor:

```clojure
(defn community-route-type
  [route-params]
  (if (string/starts-with? (:community-id route-params) "z")
    :desktop-community
    :community))
```

Reply:
Good suggestion! Thanks @ilmotta